restful_api/Controller/Service/SignoutController.py

```diff
@@ -32,4 +32,31 @@ def signout(request):
                     return Response({"message" : "token_not_match"}, status=status.HTTP_200_OK)
             else:
                 return Response({"message" : "no_account_found"}, status=status.HTTP_100_CONTINUE)
+            
```

Comment:
I'll approve this one temporarily since we will have some enhancement on our api for sign out. It will still gives our application some lazy load when we can implement an 1 API for our 3 sign out but we still implement this with different end points.

Reply:
Noted. Thankyou